tock/hours/views.py

```diff
@@ -189,9 +189,9 @@ def get_params(key):
         rev_fytd = hours_billed_fytd * target.labor_rate
 
         # Get data for reporting period and clean result.
+        rp_select_time = rp_selected.start_date.strftime('%Y-%m-%d')
         hours_billed_weekly = clean_result(TimecardObject.objects.filter(
```

Comment:
Not directly related, but I think this query could be a bit cleaner and easier to read. It just feels a bit nested. I'm not sure, but I also wonder if we might want `only()` rather than `exclude()`

Also, what's the variance in rp_selected.start_date and timecard__reporting_period__start_date that necessitates teh strftime above? Aren't they both date objects?